root/src/components/aboutUs/MeetOurTeam/MeetOurTeam.tsx

```diff
@@ -66,50 +57,49 @@ const MemberCard = ({
   socials,
   profileText,
 }: Members) => (
-  <Card>
+  <Card className="m-2 sm:m-3.5 xl:m-5">
     <a className="block group" href="#">
       {/* Card: pic + hidden card-body +  Name */}
       <div
-        className="card w-auto bg-transparent relative pl-2 py-4
+        className="card w-auto bg-transparent relative
       group-hover:card-side group-hover:mx-[30px] group-hover:rounded-8xl
       group-hover:bg-gradient-to-r group-hover:from-slate-100 group-hover:via-slate-50 group-hover:to-transparent
       dark:group-hover:bg-gradient-to-r dark:group-hover:from-dark-card2 
       group-hover:drop-shadow-lg dark:group-hover:drop-shadow-lg dark:group-hover:shadow-gray-600"
       >
         {/* Profile pic */}
-        <figure className="pl-3">
+        <figure className="pl-3 py-3 sm:py-5">
```

Comment:
I don't think we need a figure tag, we can just straight away use the img tag

Reply:
ooohhhh I followed that from daisy ui 🤣 okayyy

Reply:
oooops it looks really weird 🥲 the figure will take up a lot of spaces of the card. I've tried to change the padding around the figure into margin around the img so that there is some margin between the avatar and the border of the card, this could work... then I've tried to adjust the width of the card or the card body, but turn out if I set a fixed width for the card (in different screen width), I would need a larger width by adding the width of the photo and the card body; if I add a larger width to the card body only like in pic3, though it looks good by setting different width to different screen widths, then when I checked out Mona's picture which is the one we try automatic adjustment for picture, her picture shrinks like in pic 2 🌝 So maybe we should add a figure tag instead like p1....
<img width="1440" alt="Screen Shot 2022-05-09 at 15 37 42" src="https://user-images.githubusercontent.com/96937379/167376721-684cd9fb-58d4-4806-89af-6b3b8d577803.png">
<img width="1440" alt="Screen Shot 2022-05-09 at 17 00 16" src="https://user-images.githubusercontent.com/96937379/167376793-a445a18a-84de-4020-9f41-f8a4aea13b71.png">
<img width="1440" alt="Screen Shot 2022-05-09 at 17 00 05" src="https://user-images.githubusercontent.com/96937379/167376827-4d4e72e3-6264-499d-8dea-a239d84bf228.png">

Reply:
Let's add figure tag back. Perhaps there are custom styles of Daisy UI on the figure?

Reply:
😯 maybe...but I couldn't find any 🥲